### ISSUE TITLE
Adding keyboard shortcut as in navigator plug in

### ISSUE THREAD

**Issue by arunnbabu81** [kebifurai/TiddlyResearch#6]
Can you try adding keyboard shortcuts as in navigator plug in for 
1. editing and closing the focussed tiddler.
2. navigating to the right of the horizontal story river to the next tiddler after the focused one.
3. navigating to the left of the horizontal story river to the  previous tiddler before the focused one.
This would make using TW much faster.
I tried add the navigator plug in to tiddlyresearch, but it was showing Internal JavaScript Error.

Am attaching a copy of BTC's navigator plug in for ur reference.
[navigator.html.zip](https://github.com/kebifurai/TiddlyResearch/files/5227055/navigator.html.zip)

**Comment by postkevone** [comment]
Thanks for sharing, I'm interested in adding this feature, would you mind posting the source of the html file you uploaded?
After doing some research I found this plugin http://selectmode.tiddlyspot.com wich does pretty much the same things.

**Comment by arunnbabu81** [comment]
I got it from TW google group discussion (https://groups.google.com/g/tiddlywiki/c/y7RX8C4-YaU/m/lJlIdacvBQAJ). I don't know whether the original source is still available. BTC's GitHub page doesnt show these plug ins anymore.

**Comment by arunnbabu81** [comment]
Select mode was used in BTC's KeeBooard plug in for Muuritouch story river. I think KeeBoord plug in was replaced by navigator plug in, not sure though.

**Comment by postkevone** [comment]
Thanks for the info.
I explored the navigation plugin in detail and I noticed it hasn't been updated since 2018.
This could create a lot of incompatibilities with the current TIddlyWiki version since the plugin also modifies some core files in their integrity, so I cannot implement it just yet.
Anyway I will try to extract the code necessary for moving between tiddlers with the arrow keys for the very least.
